infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/StatusLogger.java

```diff
@@ -52,6 +52,10 @@ public void onStartup(final String version) {
             + "You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0");
   }
 
+  public void startupConfigurations(final StartupLogConfig config) {
+    config.report(log);
+  }
```

Comment:
This is kinda creating a two-way dependency that does not add any value. 

What I meant with my previous comment was considering two things:
1. that we need to use the StatusLogger (maybe we don't need it)
2. that we want a way to test the `StartupLogConfig` msg (like check we are building the expected msg/msgs.